config/webpack.config.base.js

```diff
@@ -14,20 +14,22 @@ module.exports = {
     umdNamedDefine: true
   },
   resolve: {
-    extensions: ['.js', '.json', '.yaml'],
+    extensions: ['.js', '.json', '.yaml', '.jsx'],
     modules: [SRC_DIR, 'node_modules'],
     alias: {
       react: path.resolve(__dirname, 'aliases/globalReact'),
-      'react-dom': path.resolve(__dirname, 'aliases/globalReactDOM'),
-      'cozy-ui/react': 'cozy-ui/transpiled/react'
```

Comment:
If I am not mistaken, the most important part of this commit is the removal of the alias.

Nitpick: The name of the commit is a bit confusing, we are not "transpiling", we are bundling "cozy-ui" through its non transpiled version :)

Reply:
I splitted the commit into two.